homeworks/class3/views/result.handlebars

```diff
@@ -0,0 +1,39 @@
+<body>
+	<div class="container">
+		<div class="jumbotron">
+		    <div class="container">
+				{{!-- Jumbotron Display --}}
+				{{#each cat}}
```

Comment:
You are using a lot of unnecessarily complex work on server side to only use this one handlebars, might make it easier for yourself to do more handlebars files